wagtail/admin/templates/wagtailadmin/home.html

```diff
@@ -9,13 +9,13 @@
 
 {% block content %}
     {% fragment as header_title %}
-        {% block branding_welcome %}{% blocktrans trimmed %}Welcome to the {{ site_name }} Wagtail CMS{% endblocktrans %}{% endblock %}
+        {% block branding_welcome %}{% blocktrans trimmed %}{{ site_name }} Wagtail CMS{% endblocktrans %}{% endblock %}
```

Comment:
I am curious as to the reasoning behind this change - this can be modified easily through customisations and provides a good welcome text.

Reply:
This was a change made in the design system it seems
https://www.figma.com/file/h67EsVXdWsfu38WGGxWfpi/Wagtail-Design-System?node-id=6087%3A36870

Reply:
it might be good to get some context for the change before we change it.